python/GafferUITest/GraphGadgetTest.py

```diff
@@ -1813,6 +1813,8 @@ def testDirtyTrackingForInitialFocusNode( self ) :
 		# Initially we expect the left branch of the switch to be highlighted.
 
 		window.setVisible( True )
+		self.waitForIdle( 1000 )
```

Comment:
I'm surprised that this is necessary, because `assertHighlighting()` goes to reasonable lengths to wait for the highlighting to take effect.

Reply:
I could move the increased wait count to https://github.com/GafferHQ/gaffer/blob/0c868cc20f7c72930d6ce6048342ce3a7ac9a07d/python/GafferUITest/GraphGadgetTest.py#L1784 as an alternative. I guess that the default wait count of 1 is not enough for grandpa Windows to set things in motion?

Reply:
Offline we decided to leave this in place since it's not too much of an extra delay, and the proper solution is pretty involved for a one line fix.